lib/ui/views/settings/settings_view.dart

```diff
@@ -21,6 +23,8 @@ class SettingsView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    ScreenUtil.init(context,
+        designSize: const Size(resWidth, resHeight), minTextAdapt: true);
```

Comment:
I suggest do not initialize the `screen_util` this way, adding it into the widget tree and use it wherever it's possible while adding into the widget tree from the top-level pass the design size based on that it'll adapt responsive font size.

Reply:
Fair point, definitely an oversight I did.